CK3ToEU4Tests/MapperTests/RulerPersonalityMapper/RulerPersonalitiesMappingTests.cpp

```diff
@@ -0,0 +1,70 @@
+#include "../../CK3toEU4/Source/Mappers/RulerPersonalitiesMapper/RulerPersonalitiesMapping.h"
+#include "gtest/gtest.h"
+#include <sstream>
+
+TEST(Mappers_RulerPersonalitiesMappingTests, traitsDefaultToBlank)
+{
+	std::stringstream input;
+	input << "= {}";
+
+	const mappers::RulerPersonalitiesMapping theMapping(input);
+
+	ASSERT_TRUE(theMapping.getTraits().empty());
+}
+
+TEST(Mappers_RulerPersonalitiesMappingTests, traitsCanBeLoaded)
```

Comment:
I'd drop this test and the corresponding function. The other tests prove what you want.